hardhat.config.ts

```diff
@@ -26,6 +27,17 @@ task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
   }
 });
 
+task("transfer", "Prints the list of accounts", async (taskArgs, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (let i = 9; i > 0; i--) {
+    await accounts[i].sendTransaction({
+      to: await accounts[0].getAddress(),
+      value: ethers.utils.parseEther("0.49"), // 1 ether
```

Comment:
Wrong comment

Reply:
Actually this was just a simple script to fund my deploying account because of faucet restrictions... good catch I will remove the whole task because it is not relevant